### ISSUE TITLE
Install latest version of openwhisk on kubernetes using helm

### ISSUE THREAD

**Issue by Devops-grepsr** [apache/openwhisk-deploy-kube#786]
How can we install latest version of openwhisk.
the repo consist of only old version of openwhisk.
I tried changing the commit/ image tag in values,yaml.
it didn't work.

**Comment by dgrove-oss** [comment]
Hi.  I'm sorry this project has languished, but we've unfortunately picked up a fair amount of technical debt that needs to be dealt with before we can make a decent release.  

Some of the big items are:
  1. The CI infrastructure needs to be ported from TravisCI to GitHub Actions (that's why it hasn't run in over a year).
  2. We need to update to modern versions of Kubernetes (and of Kafka, ZooKeeper, etc).
  3. The entire OpenWhisk project is behind on switching to multi-arch Docker images  (https://github.com/apache/openwhisk/issues/5503) which has really limited my personal ability to help fix this as my development machine is an arm-based Mac with rancher-desktop.

Closing as a duplicate of #781
